src/components/input-number/input-number.tsx

```diff
@@ -806,6 +806,7 @@ export class InputNumber
     };
 
     const sanitizedValue = sanitizeNumberString(
+        // no need to delocalize a string that ia already in latn numerals
```

Comment:
typo: "is" instead of "ia"

Reply:
Nice catch on the typo. Its breaking some tests which I'm looking into before merging